controllers/propagator/encryption.go

```diff
@@ -108,3 +112,42 @@ func (r *PolicyReconciler) getEncryptionKey(clusterName string) ([]byte, error)
 
 	return key, nil
 }
+
+// getInitializationVector retrieves the initialization vector from the annotation
+// "policy.open-cluster-management.io/encryption-iv" if the annotation exists or generates a new
+// initialization vector and adds it to the annotations object if it's missing.
+func (r *PolicyReconciler) getInitializationVector(
+	policyName string, clusterName string, annotations map[string]string,
+) ([]byte, error) {
+	log.WithValues("policy", policyName, "cluster", clusterName)
+
+	if initializationVector, ok := annotations[ivAnnotation]; ok {
+		log.V(2).Info("Found initialization vector annotation")
+
+		decodedVector, err := base64.StdEncoding.DecodeString(initializationVector)
+		if err != nil {
+			return nil, fmt.Errorf(
```

Comment:
As discussed over chat, it'd be nice if this error was logged and the IV was just recreated in the event a user mistakenly changed the IV.

Reply:
Do you think you could also verify the IV length is correct after it's decoded?

Reply:
That's a good idea--and then regenerate it if it's not the right length.